gsa/src/web/graphql/alerts.js

```diff
@@ -47,6 +47,7 @@ export const GET_ALERTS = gql`
           id
           inUse
           writable
+          comment
```

Comment:
Literally just adding a comment field causes tests to fail. However, this works in the UI. Comments display correctly.